bioimageio/spec/shared/_resolve_source.py

```diff
@@ -217,7 +222,18 @@ def resolve_rdf_source_and_type(
 
 
 @singledispatch  # todo: fix type annotations
-def resolve_source(source, root_path: typing.Union[os.PathLike, URI] = pathlib.Path(), output=None):
+def resolve_source(source, root_path: typing.Union[os.PathLike, URI] = pathlib.Path(), output=None, pbar=None):
+    """Resolve sources to local files
+
+    Args:
+        source: resource description as path, or uri
```

Comment:
a url to a weight file would not be a 'resource description' as in resource description file (RDF)... Of course it is not wrong to call it that, but I find this confusing
```suggestion
        source: e.g. a path or uri
```